### ISSUE TITLE
Convert Gpu Model to CPU issues and some fixes

### ISSUE THREAD

**Issue by giddyyupp** [karpathy/neuraltalk2#185]
when i tried to convert the model, i got fb.debugger errors. 
the easiest solution is to do this: 
`luarocks install https://raw.githubusercontent.com/bshillingford/fbdebugger-minimal/master/fbdebugger-standalone-1.rockspec`
this installs just the debugger not all fb libraries.
after you installed this and run the convert script, debugger comes up and it stucks, i write "quit" and finally it saved cpu model. 

so basically these are the required steps to convert a gpu model to cpu.
